### DISCUSSION TITLE
v2 Colab training error

### DISCUSSION THREAD

**Discussion by christophertrapani** [acids-ircam/RAVE#193] [Q&A]
I keep coming up with this error at the "Training" stage:

shell-init: error retrieving current directory: getcwd: cannot access parent directories: Transport endpoint is not connected
Intel MKL FATAL ERROR: Cannot load /content/miniconda/lib/python3.9/site-packages/torch/lib/libtorch_cpu.so

Indeed there doesn't seem to be a "libtorch_cpu.so" file in the designated location. 

Any ideas on what the problem is or how to fix it? Many thanks

**Comment by arpajik** [comment]
here also and training stops after determining the length with "dataset length: 3:50:59.155011: : 4664it [00:42, 110.81it/s]
FATAL Flags parsing error: Missing value for flag --ckpt
Pass --helpshort or --helpfull to see help on flags."

**Reply by arpajik** [reply]
i also ran into this error:

You currently have a PYTHONPATH environment variable set. This may cause
    unexpected behavior when running the Python interpreter in Miniconda3.
    For best results, please verify that your PYTHONPATH only points to
    directories of packages that are compatible with the Python interpreter
    in Miniconda3: /content/miniconda

if anyone is able to help, thanks a lot.

**Reply by christophertrapani** [reply]
the --ckpt error is easy to fix: you have to manually delete " --ckpt $resume" one the code of the TRAINING module

I've seen that PYTHONPATH error as well, not sure how relevant it is, but it's not FATAL: you should still be able to train

**Comment by drscotthawley** [comment]
Should the tensorboard viz be working?  I never see anything get written to the logs directory I specified.  Hence, tensorboard has nothing to show.

**Comment by serlecu** [comment]
I am getting the missing "libtorch_cpu.so" file, but when exporting as torchscript.

```
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
INFO:root:library loading
INFO:root:DEBUG
Intel MKL FATAL ERROR: Cannot load /content/miniconda/lib/python3.9/site-packages/torch/lib/libtorch_cpu.so.
```
